### DISCUSSION TITLE
乱数の固定

### DISCUSSION THREAD

**Discussion by nim-hrkn** [Tomoki-YAMASHITA/CrySPY#29] [General]
RSとEAの乱数固定はどうやるのですか。

**Comment by Tomoki-YAMASHITA** [comment]
乱数固定する方法は用意していません．
CrySPYのコード部分であれば，シード設定を入れようと思えば入れられますが
pyxtalのところがどうなるかよくわからない部分があるので今のところシードには触れていません．

乱数固定はできませんが，
RSで同じ初期構造を用いてシミュレーションを何回か行いたいという場合は
初期構造を最初に読み込むことで，同じ初期構造を再現できます．

Aディレクトリ
1. 普通にCrySPYでランダム構造生成する（例えば100構造）
2. ./data/pkl_data/init_struc_data.pkl をBディレクトリにコピー

Bディレクトリ
1. ./data/pkl_data というディレクトリを作成し，Aディレクトリのinit_struc_data.pklをここに配置
2. cryspy.inでload_struc_flag = True にする．

``` python
[basic]
tot_struc = 100
...
...
...

[option]
load_struc_flag = True
```

3. cryspyを実行するとランダム構造を生成せずに，./data/pkl_data/init_struc_data.pkl を読み込む
